documentation/staging/content/release-notes.md

```diff
@@ -6,52 +6,104 @@ draft: false
 
```

Comment:
Seems like this has gotten big enough that a table of contents could be helpful. FYI: You can generate a TOC by running `./getlinks.sh release-notes.md` from within the `documentation/staging` directory.